services/modules/helm-charts/charts/guardian-frontend/values.yaml

```diff
@@ -37,12 +37,19 @@ securityContext: {}
   # runAsUser: 1000
 
 service:
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
I'm unsure on this - I think this needs to be a LoadBalancer for AWS due to how EKS sets things up, where is this load balanced now?

Reply:
this is correct, we using ingress with will run on LoadBalancer , the front-end now just become the nginx host static website only. there is the optional var 'use_ingress' it you turn that to false, it will make this front-end become LoadBalancer